plugins/chat/plugin.rb

```diff
@@ -132,6 +132,12 @@ module ::Chat
     end
   end
 
+  add_to_serializer(:admin_plugin, :incoming_chat_webhooks) do
```

Comment:
This has to be done because chat currently hijacks the /admin/plugins/chat.json route on the server-side here:

https://github.com/discourse/discourse/blob/c7ce5a7391505b8cd0d4f83bfa273ca943cafef1/plugins/chat/app/controllers/chat/admin/incoming_webhooks_controller.rb#L8-L17

When I get to moving around stuff on the chat plugin, I will make it so there is a dedicated endpoint for the webhooks index in the config UI.